cumulus-sled

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import json
+import sys
+
+from message import message
+
+if __name__ == '__main__':
+  functionName = sys.argv[1]
+  transformer = message.message()
+  exitCode = 1
+  allInput = json.loads(raw_input())
+  event = allInput['event']
+  try:
+    if (functionName == 'loadRemoteEvent'):
+      result = transformer.loadRemoteEvent(event)
+    if (functionName == 'loadNestedEvent'):
```

Comment:
Should this be `elif`?

Reply:
Addressed in https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/b24b1d1614eade0a199f159b11f619bc5caefbe1